client/src/SubscriptionPage.js

```diff
@@ -1,135 +1,77 @@
 import { useState } from "react";
-import SubscriptionButton from "./SubscriptionButton";
+// import SubscriptionButton from "./SubscriptionButton";
 import { useNavigate } from "react-router-dom";
+import VSpacer from "./VSpacer";
 import "./SubscriptionButton.css";
 import "./SubscriptionPage.css";
 
 const SubscriptionPage = () => {
   const navigate = useNavigate();
-  const [subscriptionType, setSubscriptionType] = useState("starter");
-  const [subscriptionPrice, setSubscriptionPrice] = useState(55);
+  // const [subscriptionType, setSubscriptionType] = useState("starter");
+  // const [subscriptionPrice, setSubscriptionPrice] = useState(55);
 
-  const capitalize = (str) => str.charAt(0).toUpperCase() + str.slice(1);
+  // const capitalize = (str) => str.charAt(0).toUpperCase() + str.slice(1);
 
-  function handleSubmit() {
-    const requestOptions = {
-      method: "POST",
-      headers: { "Content-Type": "application/json" },
-      body: JSON.stringify({
-        name: `${subscriptionType} diy ikebana`,
-        amount: subscriptionPrice,
-      }),
-    };
+  // function handleSubmit() {
+  //   const requestOptions = {
+  //     method: "POST",
+  //     headers: { "Content-Type": "application/json" },
+  //     body: JSON.stringify({
+  //       name: `${subscriptionType} diy ikebana`,
+  //       amount: subscriptionPrice,
+  //     }),
```

Comment:
We can remove this since it has been implemented in the SubscriptionDetailCard

Reply:
Sweet, thanks - I just removed it